requirements/deploy.txt

```diff
@@ -4,7 +4,10 @@
 # http://initd.org/psycopg/docs/install.html#binary-install-from-pypi
 psycopg2==2.8.3 --no-binary psycopg2
 gunicorn==20.0.4
-django-redis-cache==2.1.3
+
+# Version 3.0.0 drops support for Django < 3.0
+# https://github.com/sebleier/django-redis-cache/#300
+django-redis-cache==2.1.2  # pyup: ignore
```

Comment:
```suggestion
django-redis-cache==2.1.3  # pyup: ignore
```

2.1.3 is the current version in our requirements

Reply:
Ups...